pycbc/io/hdf.py

```diff
@@ -960,13 +960,53 @@ def recursively_save_dict_contents_to_group(h5file, path, dic):
         python dictionary to be converted to hdf5 format
     """
     for key, item in dic.items():
-        if isinstance(item, (np.ndarray, np.int64, np.float64, str, bytes, tuple, list)):
+        if isinstance(item, (np.ndarray, np.int64, np.float64, str, int, float,
+                             bytes, tuple, list)):
             h5file[path + str(key)] = item
         elif isinstance(item, dict):
             recursively_save_dict_contents_to_group(h5file, path + key + '/', item)
         else:
             raise ValueError('Cannot save %s type' % type(item))
 
+def load_hdf5_to_dict(filename):
```

Comment:
Do you actually use this function? If not, please remove.

Reply:
I wrote it this way to mimic the two already existing functions `recursively_save_dict_contents_to_group` and `save_dict_to_hdf5`,  but it's simpler as you say